packages/image_picker/LICENSE

```diff
@@ -24,3 +24,17 @@ DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
 THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+--------------------------------------------------------------------------------
+Copyright 2011 - 2013 Paul Burke
```

Comment:
This is *not* the content of https://raw.githubusercontent.com/iPaulPro/aFileChooser/master/LICENSE.txt?

/cc @Hixie

Reply:
[See my previous comment for the reasoning about this.](https://github.com/flutter/plugins/pull/423#discussion_r175929999)

Reply:
This must be the contents of <https://raw.githubusercontent.com/iPaulPro/aFileChooser/master/LICENSE.txt>.